editoast/src/client/mod.rs

```diff
@@ -117,6 +118,13 @@ pub struct ImportProfileSetArgs {
     pub electrical_profile_set_path: PathBuf,
 }
 
+#[derive(Args, Debug)]
+#[command(about, long_about = "Import a rolling stock given a json file")]
+pub struct ImportRollingStockArgs {
+    /// Rolling stock file path
+    pub rolling_stock_path: PathBuf,
```

Comment:
We could allow multiple source files.

```suggestion
    pub rolling_stock_paths: Vec<PathBuf>,
```